packages/core/src/view/canvas/HtmlCanvas2D.ts

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright 2021-present The maxGraph project Contributors
+Copyright (c) 2006-2015, JGraph Ltd
+Copyright (c) 2006-2015, Gaudenz Alder
```

Comment:
Original maxGraph code
```suggestion
Copyright 2023-present The maxGraph project Contributors
```